src/submit_jobs.py

```diff
@@ -246,9 +246,8 @@ def read_job(job: SparkJob):
         github = GitHubClient(gh_pem_path=PATH_GH_PEM)
         github.run_workflow(
             repository=(github.gh_api_url + "ccao-data/data-architecture"),
-            workflow="test_dbt_models.yaml",
+            workflow="test_iasworld_data.yaml",
             inputs={
-                "selector": "select_data_test_iasworld",
```

Comment:
We no longer need to specify the selector, since the `test-iasworld-data` workflow defaults to running all iasWorld tests.